README.md

```diff
@@ -1,19 +1,38 @@
-# Kinde Elixir SDK
+# Kinde Elixir
 
 The Kinde SDK for Elixir.
 
-You can also use the Elixir starter kit [here](https://github.com/kinde-starter-kits/elixir-starter-kit).
+You can also use the Elixir starter kit [here](https://github.com/kinde-starter-kits/elixir-starter-kit).
 
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://makeapullrequest.com) [![Kinde Docs](https://img.shields.io/badge/Kinde-Docs-eee?style=flat-square)](https://kinde.com/docs/developer-tools) [![Kinde Community](https://img.shields.io/badge/Kinde-Community-eee?style=flat-square)](https://thekindecommunity.slack.com)
 
+### Initial set up
```

Comment:
Thanks for the contribution!

For clarity and better usability, it's usually best to start with the integration instructions. Therefore, this section should ideally be after the documentation section. It might also be helpful to title it "Development Setup" to clearly indicate its purpose.

Thank you for your effort on this!